solutionbox/inception/datalab_solutions/inception/_cloud.py

```diff
@@ -20,18 +20,18 @@
 import base64
 import collections
 import datetime
-from googleapiclient import discovery
 import google.cloud.ml as ml
 import logging
 import os
 
+
 from . import _model
 from . import _preprocess
 from . import _trainer
 from . import _util
 
 
-_TF_GS_URL= 'gs://cloud-datalab/deploy/tf/tensorflow-0.12.0rc0-cp27-none-linux_x86_64.whl'
+_TF_GS_URL= 'gs://cloud-datalab/deploy/tf/tensorflow-0.12.0rc1-cp27-none-linux_x86_64.whl'
```

Comment:
We could. But I am not sure how long they will keep these releases there. It seems better to store dependencies in our own bucket, as much as possible?